pa11ycrawler/templates/result.html

```diff
@@ -7,7 +7,7 @@
 
 <div class="col-sm-10 col-sm-offset-1" style="margin-bottom: 30px;">
     <h1>Accessibility Audit for ${info['page_title']}</h1>
-    <p><span class='h5'>URL</span> ${url}</p>
+    <p><a href="${url}">${url}</a></p>
```

Comment:
This is great.  If we want to demonstrate best practices (we should probably eat our own dogfood here) we don't want the value of the link text to be the URL (terrible to hear as a screen reader user).  Can we change the link text to be "View page used to generate this report"?